stylesheets/components/Inputs/Input.css

```diff
@@ -0,0 +1,177 @@
+
+
+/* Input Field */
+.input--field {
+  height: 56px;
+  width: 344px;
+  padding: 12px;
+  border-radius: 8px;
+  border: 3px solid var(--component-grey-01--hover);
+  background-color: hsla(0, 0%, 88%, 50%);
+  color: var(--component-grey-05);
+}
+
+.input--field::placeholder {
+  color: var(--component-grey-03);
+}
+
+.input__container {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
You can extract it in a utility class

Reply:
The justify property was supposed to be flex-start actually. I'll try extracting it during refactoring.